web/loadChunkWorker.js

```diff
@@ -0,0 +1,33 @@
+self.addEventListener('message', async (e) => {
```

Comment:
Basically exactly the same as the old LoadChunk method, except a little more optimized and with an additional sorting that happens if user data is loaded